.github/workflows/uberjar.yml

```diff
@@ -73,3 +73,63 @@ jobs:
           ./target/uberjar/metabase.jar
           ./COMMIT-ID
           ./SHA256.sum
+
+  check:
+    runs-on: ubuntu-20.04
+    needs: build
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        edition: [ee, oss]
+        java-version: [8, 11, 13]
+    steps:
+    - name: Prepare JRE (Java Run-time Environment)
+      uses: actions/setup-java@v1
+      with:
+        java-package: jre
+        java-version: ${{ matrix.java-version }}
+    - run: java -version
+
+    - uses: actions/download-artifact@v2
+      name: Retrieve uberjar artifact
+      with:
+        name: metabase-${{ matrix.edition }}-uberjar
+
+    - name: Launch uberjar
+      run: |
+        java -jar ./target/uberjar/metabase.jar &
+        sleep 180
+      timeout-minutes: 5
+
+    - name: Check API health
+      run: curl -s localhost:3000/api/health
```

Comment:
very cool! is there another super basic health test we can do (like load the main page of metabase to get a 200) or is the plan to follow on with more steps in subsequent PRs?

Reply:
Definitely we can slowly decide and add more tests.
I believe the focus should be tests which quickly identify any problem due to the test variants (in this case, JRE versions, and as illustration in our code not supporting Java 15 yet).